resources/quality/ultimaker_s3/um_s3_aa0.4_BAM_VeryDraft_Print.inst.cfg

```diff
@@ -0,0 +1,41 @@
+[general]
+version = 4
+name = Extra Fast
+definition = ultimaker_s3
+
+[metadata]
+setting_version = 19
+type = quality
+quality_type = verydraft
+weight = -2
+material = generic_bam
+variant = AA 0.4
+is_experimental = True
+
+[values]
+brim_replaces_support = False
+cool_fan_full_at_height = =layer_height_0 + 2 * layer_height
+cool_fan_speed_max = =cool_fan_speed
+machine_nozzle_cool_down_speed = 0.75
+machine_nozzle_heat_up_speed = 1.6
+material_print_temperature = =default_material_print_temperature + 5
+# prime_tower_enable: see CURA-4248
```

Comment:
Ah, sorry, can't have these comments in here. Our fast parser doesn't deal with those safely.